### ISSUE TITLE
Crash after scan document: EXC_BAD_ACCESS

### ISSUE THREAD

**Issue by olcayertas** [regulaforensics/DocumentReader-iOS#29]
Application crashing after scanning a document with following stact trace:

```
objc_release 0x00000001af12c140
__50-[RGLDocumentReaderCameraViewController scanImage]_block_invoke_11 0x0000000100d61a94
_dispatch_call_block_and_release 0x00000001af0b6610
_dispatch_client_callout 0x00000001af0b7184
_dispatch_main_queue_callback_4CF$VARIANT$armv81 0x00000001af09a34c
__CFRUNLOOP_IS_SERVICING_THE_MAIN_DISPATCH_QUEUE__ 0x00000001af3685e4
__CFRunLoopRun 0x00000001af3635d8
CFRunLoopRunSpecific 0x00000001af362adc
GSEventRunModal 0x00000001b92e8328
UIApplicationMain 0x00000001b345dae0
main main.swift:8
start 0x00000001af1ec360
```
Error:
`Exception: EXC_BAD_ACCESS (code=1, address=0x4a7aa6c20)`

**Comment by vyakimchik** [comment]
Hello. Please, provide the steps on how to reproduce the issue. If the app crashes while you are scanning the specific document, please, send it to dev.support@regulaforensics.com. Thanks!

**Comment by olcayertas** [comment]
It doesn't crash for a particular document every time but we got an other crash while scanning a document this time stack trace of crash report looks like this:
<img width="567" alt="Screen Shot 2020-01-23 at 09 59 52" src="https://user-images.githubusercontent.com/1506019/72963027-23679a80-3dc7-11ea-8973-79507ce61b21.png">

**Comment by vyakimchik** [comment]
Hello. We published the new version (5.1). Please check if the issue can be reproduced there.
